_includes/moto.md

```diff
@@ -0,0 +1,102 @@
+{% if include.api_resource == "creditcard" %}
+    {% assign api_resource_field_name = "payment" %}
+    {% assign api_redirect_rel = "authorization" %}
+    {% assign api_redirect_link = "creditcard/payments/authorize" %}
+{% else %}
+    {% assign api_resource_field_name = "paymentorder" %}
+    {% assign api_redirect_rel = "paymentorder" %}
```

Comment:
Is the name of the `rel` really `redirect-paymentorder`?

Reply:
yes, I got this confirmed with @sigridbra

Reply:
Yes 😅